### ISSUE TITLE
Remove hard-coded byte-sequence len for pos/vel/acc arrays

### ISSUE THREAD

**Issue by gavanderhoorn** [ros-industrial/packet-simplemessage#13]
This was (re)introduced in 719fd51a as part of the implementation of the *don't display invalid fields* configuration item (for #6). Code now (again) assumes 10 items per array, with a byte size of 4 bytes per item. This holds for now, but could change.

**Comment by simonschmeisser** [comment]
Would be nice to at least have a single constant to change when going from 10 to 20 (yumi)

**Comment by gavanderhoorn** [comment]
If you are still interested I would welcome a PR. Otherwise I'll add it to my TODO list, but no guarantees when/if that gets processed.

**Comment by simonschmeisser** [comment]
I switched to using the default abb_driver on this yumi so it's back to 10 joints. Also my wireshark skills were to weak to actually get this to run :( So no PR in the near future I guess

**Comment by gavanderhoorn** [comment]
> Also my wireshark skills were to weak to actually get this to run :(

are you saying that you could not get the plugin to work at all?

**Comment by simonschmeisser** [comment]
yes, but I'm not implying that anything is broken necessarily ... I'm just not very skilled in using wireshark

**Comment by gavanderhoorn** [comment]
Off-topic for this issue, but: installation is really just copying the file to the location indicated in the `readme`. After that Wireshark should automatically start using the plugin for any traffic on the registered ports (`11000, 11002, 50240, 50241`). No other configuration is necessary.
